vpd-manager/include/utility/vpd_specific_utility.hpp

```diff
@@ -552,5 +552,35 @@ inline void resetDataUnderPIM(const std::string& i_objectPath,
                             " with error: " + std::string(l_ex.what()));
     }
 }
+
+/**
+ * @brief API to notify FRU VPD Collection status.
+ *
+ * This API uses PIM's Notify method to update the given FRU VPD collection
+ * status on D-bus.
+ *
+ * @param[in] i_inventoryPath - D-bus inventory path
+ * @param[in] i_fruCollectionStatus - FRU VPD collection status.
+ *
+ * @return true if update succeeds, false otherwise.
+ */
+inline bool notifyFRUCollectionStatus(const std::string& i_inventoryPath,
```

Comment:
This API just updates a property on DBus. 
Should this be under dbusUtility?

Reply:
moved it to dbusUtility